src/Microsoft.IdentityModel.Tokens/opensource/json/JsonTextWriter.Async.cs

```diff
@@ -131,7 +134,34 @@ internal async Task DoCloseAsync(CancellationToken cancellationToken)
                 await WriteEndAsync(cancellationToken).ConfigureAwait(false);
             }
 
-            CloseBufferAndWriter();
+            await CloseBufferAndWriterAsync().ConfigureAwait(false);
+        }
+
+        private async Task CloseBufferAndWriterAsync()
+        {
+            if (_writeBuffer != null)
+            {
+                BufferUtils.ReturnBuffer(_arrayPool, _writeBuffer);
+                _writeBuffer = null;
+            }
+
+            if (CloseOutput && _writer != null)
+            {
+#if HAVE_ASYNC_DISPOABLE
```

Comment:
>DISPOABLE [](http://example.com/codeflow?start=15&length=9)

spelling: DISPOSABLE  -> missing an S

Reply:
https://github.com/JamesNK/Newtonsoft.Json/pull/2698

Reply:
I changed it, to be consistent with the repo, but the define in M.IM.Tokens.csproj was also misspelled.
So, i changed them both.